zeapp/badge/src/windowsMain/kotlin/de/berlindroid/zekompanion/BadgeManager.kt

```diff
@@ -0,0 +1,17 @@
+package de.berlindroid.zekompanion
+
+import com.fazecast.jSerialComm.SerialPort
+
+
+actual typealias Environment = Any
+
+actual fun buildBadgeManager(environment: Environment): BadgeManager = object : JvmBadgeManager() {
+    override fun getBadger2040s() = SerialPort.getCommPorts()
+        .filter {
+            println(it.systemPortName)
+            true
+            // return the first serial comm device, can't find it by descriptive name
+//            it.descriptivePortName.contains("Badger")
```

Comment:
I feel that this doesn't filter anything. Maybe it's a leftover debug statement?

Please remove the comments and don't print something, it feels more noisy then we need.

Reply:
this is mostly reason why this is still draft, the device is not called Badger on Windows

Reply:
Updated it with the best possible identifier I could find
![Screenshot 2024-04-20 141305](https://github.com/gdg-berlin-android/ZeBadge/assets/1077945/f9edb28c-c3fe-44e3-925e-a17a96a79ea5)